### PR TITLE
MCR-1912 add messages for better logging

### PR DESCRIPTION
**Pull request by bt1cu**
[Link to jira](https://mycore.atlassian.net/projects/MCR/issues/MCR-1912).

### CODE REVIEW

**Review by rsteph-de — APPROVED** [review]
I like it.

File: mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java [hunk 1]
```diff
@@ -147,21 +148,22 @@ private static void executeSolrCommand(String coreID, String command) throws Uns
...
         post.setHeader("Content-type", "application/json");
         post.setEntity(new StringEntity(command));
+        String commandprefix = command.indexOf('-') != -1 ? command.substring(2,command.indexOf('-')) : "unknown command";
         HttpResponse response;
         try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
             response = httpClient.execute(post);
             String respContent = new String(ByteStreams.toByteArray(response.getEntity().getContent()),
                 StandardCharsets.UTF_8);
             if (response.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
-                LOGGER.debug("SOLR config update command was successful \n" + respContent);
```
Comment (paschty) [inline mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java]: Better use: `LOGGER.debug("SOLR config {} command was successful \n" , respContent);`

File: mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java [hunk 2]
```diff
@@ -147,21 +148,22 @@ private static void executeSolrCommand(String coreID, String command) throws Uns
...
+                LOGGER.debug("SOLR config " + commandprefix + " command was successful \n" + respContent);
```
Comment (sebhofmann) [inline mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java:158]: You could use:
`LOGGER.debug("SOLR config {} command was successful \n {}",commandprefix, respContent);`

Comment (bt1cu) [reply mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java:158]: thank you

**Pull request merged**